# SBG Ellipse Messages
Header header

# Time since sensor is powered up us 
uint32 time_stamp

# Pressure status (True if the altimeter equipment was correctly initialized)
bool valid_pressure

# Altitude (True if the altitude output is valid)
bool valid_altitude

# Pressure measured by the sensor (Pa)
float32 pressure

# Altitude computed from altitude (m)
float32 altitude
